lib/sql.js

```diff
@@ -912,6 +971,42 @@ SQLConnector.prototype.buildFields = function(model, data, excludeIds) {
 };
 
 /**
+ * Build an array of fields for the replace database operation
+ * @param {String} model Model name
+ * @param {Object} data Model data object
+ * @param {Boolean} excludeIds Exclude id properties or not, default to false
+ * @returns {{names: Array, values: Array, properties: Array}}
+ */
+SQLConnector.prototype.buildReplaceFields = function(model, data, excludeIds) {
```

Comment:
I think this is an internal/private method that should not be called by connector users. If that's correct, then the method name should be prefixed with underscore, i.e. `_buildReplaceFields`.